src/components/Game/PhraseScrambler/PhraseScramblerData.js

```diff
@@ -56,16 +50,54 @@ function PhraseScramblerData() {
   const resetGame = () => {
     // Reset game state
     setSelectedWords([])
-
     // Should we also reset the jumbled words here ?
```

Comment:
I would suggest not re-jumbling the words, if the goal is to reset the game to it's original state.